vars/sectionDeployToEnvironment.groovy

```diff
@@ -60,7 +60,7 @@ def call(params) {
                   
                   builder.dbMigrate(
                     tfOutput.vaultName ? tfOutput.vaultName.value : "${config.dbMigrationVaultName}-${environment}",
-                    tfOutput.microserviceName? tfOutput.microserviceName.value : component
+                    config.uniqueDbMigrateSecretNames? "${product}-${component}" : component
```

Comment:
would rather auto prefix with PR number if on preview, but let's have a chat about this change?

Reply:
sure